### PR TITLE
Fixed author images not converting to string correctly

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

File: LDSContent/ItemPackage.swift [hunk 1]
```diff
@@ -317,7 +317,7 @@ extension ItemPackage {
         static let id = Expression<Int64>("_id")
         static let givenName = Expression<String>("given_name")
         static let familyName = Expression<String>("family_name")
-        static let imageRenditions = Expression<String>("image_renditions")
+        static let imageRenditions = Expression<String?>("image_renditions")
```
Comment (hiltonc) [inline LDSContent/ItemPackage.swift]: Can we treat an empty string as no renditions instead?

Comment (stephanheilner) [inline LDSContent/ItemPackage.swift]: We could. I was just mirroring the way you'd done image renditions everywhere else.

Comment (hiltonc) [inline LDSContent/ItemPackage.swift]: Sorry :) I think I did it wrong everywhere else.

Comment (stephanheilner) [inline LDSContent/ItemPackage.swift]: Alrighty.... I can make the change.

File: LDSContent/Author.swift [hunk 2]
```diff
@@ -27,10 +27,13 @@ public struct Author: Equatable {
     public let id: Int64
     public let givenName: String
     public let familyName: String
-    public let imageRenditions: [ImageRendition]
+    public let imageRenditions: [ImageRendition]?
```
Comment (hiltonc) [inline LDSContent/Author.swift]: Can this be not optional, and just treat an empty array as no renditions?

Comment (hiltonc) [inline LDSContent/Author.swift]: I may be missing something in how this library is used?

**Pull request merged**